ui/components/section/section.go

```diff
@@ -37,6 +37,7 @@ type Model struct {
 	PromptConfirmationBox     prompt.Model
 	IsPromptConfirmationShown bool
 	PromptConfirmationAction  string
+	LastTaskId                string
```

Comment:
Can we rename this to `LastFetchTaskId` to make it clearer?

Reply:
@dlvhdr 
Thank you for the suggestion. I've fixed.